### ISSUE TITLE
Merge work with similar project?

### ISSUE THREAD

**Issue by davenquinn** [cecgeology/USGS-FGDC#1]
I just stumbled across this repository; it is a great contribution. I've been working on [a similar library](https://davenquinn.com/projects/geologic-patterns/) over the past few years, and have gotten some outside help in curating from the USGS and others. I think centralizing the basic symbols might be useful to avoid duplication of effort.

It would also be nice to have scripts for color changing etc., and provide a single location to aggregate derived or associated symbols (e.g. the entire map symbology for linework is rather untouched as yet).

Would you be open to evaluating overlap with the [geologic-patterns](https://github.com/davenquinn/geologic-patterns) repository, and potentially create a set of tools referencing these patterns? I'd like to figure out a strategy for building a functional and growing collection of geologic symbology; having other interested parties involved in this effort would be great!

**Comment by cecgeology** [comment]
Hi,
I just saw your comment. What do you think the overlap is? I converted all the Otago converted symbols and they can all also be changed to any brush color from within QGIS so that you don't need a script to do that.
It might make sense to have less duplication, but it seems your set is not specifically for QGIS in that regard?

**Comment by davenquinn** [comment]
I think the overlap is that, to the extent that there are FGDC symbols used in both projects, it might be nice to have the symbols themselves (i.e. the SVG files) stored centrally, and then used in various other tools for different platforms like QGIS. For patterns, we've gotten a few fixes added from the USGS to solve minor graphical bugs. A well-curated and centralized set of linework symbols is still needed.

**Comment by cecgeology** [comment]
Yes, I think you are correct on the overlap. It would be best to have a central location.  (finally go to this)
I spot checked a few svg across both, and the ones I have in this repository have much simpler/more compact xml as they were simplified at some point, for example all the same fill and stroke lines get grouped so there is only one fill/stroke per file instead of per path. 
This header to start each stroke:
     fill="none"
     stroke="param(outline) #231f20"
     stroke-width=".3"
versus: 
<line style="fill:none;stroke:#000000;stroke-width:0.3;" x1="35.973" y1="-31.243" x2="42.342" y2="-31.243"/>

In the past the second one did not work in QGIS to change the colors, but it seems to now, I think. I don't know how the parser works so I don't know which is computationally more efficient.
can your applications handle the param() declaration?
Maybe It might be easiest to centralize the fixed color ones and have the QGIS parameterized ones pulled out separately?
Yes, the line styles need to be fixed too. I found some here: https://sourceforge.net/projects/qgisgeologysymbology/files/
But, it looks like these guys have superseded me:
https://github.com/BC-Consulting/FGDC-4-QGIS
